brion/detail/meshHDF5.h

```diff
@@ -75,174 +78,167 @@ class MeshHDF5 : public Mesh
     MeshHDF5(const std::string& source, const bool overwrite,
              const MeshVersion version)
         : Mesh(source, version)
-        , _file()
+        , _file([source, overwrite] {
+            try
+            {
+                SilenceHDF5 silence;
```

Comment:
silence is not part of highfive now?

Reply:
It was merged on Friday afternoon, that why I didn't use it yet.